scripts/i18n/update-source.js

```diff
@@ -1,15 +1,19 @@
 const log4js = require(`log4js`)
 const shell = require(`shelljs`)
+const path = require(`path`)
 
 let logger = log4js.getLogger(`update-source`)
 
-const host = `https://github.com`
+const protocol = `https://`
+const host = `github.com`
 const cacheDir = `.cache`
 const owner = `gatsbyjs`
 // Repo to be used as basis for translations
 const sourceRepo = `gatsby-i18n-source`
 
-const sourceRepoUrl = `${host}/${owner}/${sourceRepo}.git`
+const sourceRepoUrl = `${protocol}${process.env.GITHUB_API_TOKEN}@${host}/${owner}/${sourceRepo}.git`
```

Comment:
Could we make this either `GITHUB_ADMIN_API_TOKEN` or `GITHUB_BOT_API_TOKEN` to make it consistent? I make the distinction since there's some stuff we only want admin access on and some stuff we want the bot to do. I think it makes sense for the bot to push this stuff, but I guess that means we need to make it an owner of this repo?

Reply:
`GITHUB_API_TOKEN` is already available on Circle as an env variable (used for starter mirroring) 

We could definitely move to `GITHUB_BOT_API_TOKEN` though (since the one that is available _is_ a bot token)